components/card/card.css

```diff
@@ -0,0 +1,129 @@
+
+:root{
+   --orange:orange;
+   --gold:gold;
+   --orange-dark:rgb(255, 123, 0);
+   --red:red;
+   --white:white;
+   --pink:pink;
+   --grey:grey;
+
+}
+
+.card {
+    position:relative;
+    margin-top: 20px;
+    position: relative;
+    height: 24rem;
+    width: 12rem;
+    border-radius: 10px;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 28px,
+      rgba(0, 0, 0, 0.22) 0px 10px 10px;
+  }
+  .sale-badge {
+      padding-top:3px;
+      font-weight:500;
+    position: absolute;
+    top: -3px;
+    left: -5px;
+    height: 1.7rem;
+    width: 4.7rem;
+    text-align: center;
+    border: 1px solid var(--orange);
+    background-color: (--gold);
+    border-radius: 10px;
+  }
+  .card-image {
+    border-radius: 5px 5px 0px 0px;
+    height: 14rem;
+    width: 12rem;
+  }
+  .card-text h3{
+      font-size: 20px;
+      padding-bottom:8px;
+  }
+  .card-text {
+    font-size: 13px;
+    margin: 0px 10px 0 10px;
+    padding:10px;
+  }
+  
+  .footer {
+    margin-top: 0px;
+    padding-top: 0px;
+    display: flex;
+    justify-content: space-around;
+    align-items: baseline;
+  }
+  .footer button{
+    border:none;
+    color:var(--white);
+    background-color:var(--red);
+    height:2rem;
+    width:3.5rem;
+    border-radius:5px;
+  }
+  .footer button:hover{
+    background-color:var(--orange-dark);
+  }
+  
+  .dismiss-badge{
+    font-size:20px;
+    font-weight:bold;
+      position: absolute;
+    top:0px;
+    right:0px;
+    width: 2rem;
+    color:var(--red);
+    text-align: center;
+    border:2px var(--red) solid;
+    background-color: var(--white);
+    border-radius:3px;
+      box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 28px,
+      rgba(0, 0, 0, 0.22) 0px 10px 10px;
+  }
+  .dismiss-badge:hover{
+    background-color:var(--pink);
+    
+  }
+  .text-overlay{
+    font-weight:bold;
+    font-size:26px;
+    padding-top:6rem;
+    text-align:center;
+    background-color:(--grey);
```

Comment:
var here also, it will make the CSS code more consistent

Reply:
fixed